app/src/ui/app.tsx

```diff
@@ -981,7 +981,7 @@ export class App extends React.Component<IAppProps, IAppState> {
     return repository.gitHubRepository.htmlURL
   }
 
-  private openCurrentRepositoryInShell() {
+  private openCurrentRepositoryInShell = () => {
```

Comment:
We have a few places inside `App` where we've not used this syntax to ensure it gets bound, but without this change I can't use this in #7117 because tslint warns me:

```
ERROR: /Users/shiftkey/src/desktop/app/src/ui/app.tsx[1598, 36]: A class method is passed as a JSX attribute without having the 'this' reference bound in the constructor: this.openCurrentRepositoryInShell
```

Reply:
i'm all for this :)